birdhouse/components/finch/default.env

```diff
@@ -6,6 +6,7 @@
 
 export FINCH_VERSION=0.9.2
 export FINCH_IMAGE="birdhouse/finch:version-${FINCH_VERSION}"
+export FINCH_IMAGE_URI="registry.hub.docker.com/${FINCH_IMAGE}"
```

Comment:
In addition to be in `DELAYED_EVAL` list, all xxx_IMAGE and xxx_IMAGE_URL have to be in single quote for delayed eval to work properly.

Reply:
Yes, I noticed after. In the process of rewriting with adjustment to unittests evaluating them as well.